src/tests/mod.rs

```diff
@@ -5,3 +5,4 @@ pub mod reachability;
 pub mod refinement;
 pub mod sample;
 pub mod save_component;
+pub mod EdgeID;
```

Comment:
Change the folder name EdgeID to snake case

Reply:
fixed